functions/packages.bash

```diff
@@ -3,15 +3,14 @@
 samba_setup() {
   echo -n "$(timestamp) [openHABian] Setting up Samba network shares... "
   if ! command -v samba &>/dev/null; then
-    cond_redirect apt-get update
```

Comment:
theoretically a user could land here without apt-get update being run but usually he would choose the "update" menu option before anyway sooner or later and risk is minimal because it only has impact on packages that have changed since last update